### DISCUSSION TITLE
AI assistant

### DISCUSSION THREAD

**Discussion by ay4t** [basoro/Khanza-Lite#288] [Ideas]
Apakah berminat untuk di bantu ditambahkan AI assistant pak?

Ide fitur:
- jika adalah pasien lama AI dapat memberikan informasi saran yang relevan berdasarkan riwayat penyakit pasien yang pernah diinput

**Comment by basoro** [comment]
Wah mantap ini idenya?
Konsepnya gimana ya?
Atau ada contohnya...

Salam
Basoro

**Reply by ay4t** [reply]
ada pak , kemarin saya iseng buat UI untuk mau presentasi tetapi teman saya katanya bakal tidak terlalu dibutuhkan, seperti jadi useless. 

coba saya upload dulu pak 
@basoro 

kita bisa menggunakan model local saya coba dlu pak
![2024-12-02-18-19-19](https://github.com/user-attachments/assets/00b18a5a-7587-4881-ae44-80a249788713)

**Reply by ay4t** [reply]
![image](https://github.com/user-attachments/assets/a685c70b-f53d-436d-8d18-5e368aa2da28)

**Reply by basoro** [reply]
Kalo ini berbasis PHP, kayaknya seru kalo bisa diintegrasikan...
Paling tidak sebagai second opinion dalam pengambilan keputusan medis..

Gak sabar nunggu contoh realnya...
Mungkin bisa forking mLITE dan pull request untuk dicoba sama2

Salam 
Basoro

**Reply by ay4t** [reply]
bisa pak di php juga sama saja. 
akan saya coba di local dlu pak

**Reply by ay4t** [reply]
kalau boleh tahu ini project non-profit apa komersil ya pak ?

**Reply by basoro** [reply]
Project not profit pak...
Info lanjutan bisa disini mlite.id

Salam
Basoro

**Reply by basoro** [reply]
Ada pencerahan mas ide integrasi AI nya?

**Reply by ay4t** [reply]
ada pak, tapi saat ini masih mencari kerjasama yang komersil dlu pak 🙏
@basoro

**Reply by basoro** [reply]
Baik pak
